emtable/metadata.py

```diff
@@ -575,6 +574,14 @@ def _getFormatStr(v):
     return '.6f' if isinstance(v, float) else ''
 
 
+def _split(string, shlex=False):
+    if shlex:
+        import shlex
```

Comment:
I thought that @azazellochg commented about this. This import should be at the top level if shlex is already part of the standard library and does not introduce any problematic dependency

Reply:
I'll fix it